packages/quantic/cypress/integration/facets/facet/facet-assertions.ts

```diff
@@ -0,0 +1,31 @@
+export interface ComponentSelector {
+  get: () => Cypress.Chainable<JQuery<HTMLElement>>;
```

Comment:
CypressSelector here too?

Reply:
oops! This file should not be there anymore